test/helpers/set-test-env.ts

```diff
@@ -158,6 +162,26 @@ const fundMarket = async (testEnv: TestEnv) => {
       amount: bnedAmount,
       hre,
     })
+
+    // Get funds for deployer for bonus int
+    await getFunds({
+      to: deployer,
+      tokenSym: await token.symbol(),
+      amount: bnedAmount,
+      hre,
+    })
+
+    // Approve protocol for bonus int
+    await token
+      .connect(deployer)
+      .approve(tToken.address, bnedAmount.mul(10))
+      .then(({ wait }) => wait())
+
+    const allowance = await token.allowance(
+      await deployer.getAddress(),
+      tToken.address
+    )
```

Comment:
this needs to be moved to the `market` deploy script so that the allowance is set on deployment, not just the tests.

Will need to add a network check to prevent calling `getFunds` on any non-`hardhat` networks.

Also, only set allowance if the current `allowance === 0`